### PR TITLE
Change /etc/inmanta.d/ --> /etc/inmanta/inmanta.d/

### PR DESCRIPTION
**Pull request by arnaudsjs**

### CODE REVIEW

File: docs/administrators/auth.rst [hunk 1]
```diff
@@ -242,7 +242,7 @@ issued by keycloak.
    Show the correct configuration parameters in JSON format.
 
 Add a keycloak configuration parameters to the dashboard section of the server configuration file.
-(/etc/inmanta/server.cfg in most installs.) This section should already contain enabled=true and the
+(/etc/inmanta/inmanta.d/<custom-config>.cfg in most installs.) This section should already contain enabled=true and the
```
Comment (wouterdb) [inline docs/administrators/auth.rst]: add code block with the example section in it?

Comment (wouterdb) [reply docs/administrators/auth.rst]: and perhaps name the file /etc/inmanta/inmanta.d/dashboard.cfg

File: docs/install.rst [hunk 2]
```diff
@@ -190,8 +190,8 @@ Restart the PostgreSQL server to apply the changes made in the ``pg_hba.conf`` f
 Step 4: Set the database connection details
 -------------------------------------------
 
-Adjust the ``/etc/inmanta/server.cfg`` file as such that it contains the correct database connection details. Add/Change the
-database section of that file in the following way:
+Add a ``/etc/inmanta/inmanta.d/<custom-config>.cfg`` file as such that it contains the correct database connection details.
```
Comment (wouterdb) [inline docs/install.rst]: give the file a concrete name

File: docs/administrators/auth.rst (hunk 1 above)
Comment (arnaudsjs) [reply docs/administrators/auth.rst]: I didn't add the requested code block. It was already present in the file.

**Pull request merged**